client-app/shared/wishlists/components/wishlist-line-items.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="vc-wishlist-line-items">
+    <!-- table header -->
+    <div class="vc-wishlist-line-items__header">
+      <div class="vc-wishlist-line-items__product">
+        {{ $t("shared.wishlists.wishlist_line_items.product") }}
+      </div>
+      <div class="vc-wishlist-line-items__properties">
+        {{ $t("shared.wishlists.wishlist_line_items.properties") }}
+      </div>
+      <div class="vc-wishlist-line-items__price">
+        {{ $t("shared.wishlists.wishlist_line_items.price_per_item") }}
+      </div>
+      <div class="vc-wishlist-line-items__quantity">
+        {{ $t("shared.wishlists.wishlist_line_items.quantity") }}
+      </div>
+      <div class="vc-wishlist-line-items__remove-button"></div>
+    </div>
+
+    <!-- table body -->
+    <div v-if="items.length" class="vc-wishlist-line-items__body">
+      <WishlistLineItem v-for="item in items" :key="item.id" :item="item" @remove="$emit('item:remove', item)" />
+    </div>
+
+    <div v-else class="vc-wishlist-line-items__empty">
+      <VcAlert type="warning" icon>
+        {{ $t("shared.wishlists.wishlist_line_items.no_items_message") }}
+      </VcAlert>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { PropType } from "vue";
+import { LineItemType } from "@/xapi";
+import { WishlistLineItem } from "@/shared/wishlists";
+
+defineProps({
+  items: {
+    type: Array as PropType<LineItemType[]>,
+    required: true,
+  },
+});
+
+defineEmits(["item:remove"]);
```

Comment:
to use event names by analogy as here: https://github.com/VirtoCommerce/vc-theme-b2b-vue/pull/421/files#diff-bb861bf4e0e94866ec6310e0fcdc2bf4fc68f0225d5822b9497bb9845e29c98cR75-R78

`@<action>:<object>:<object_property>`